docs/source/streamflow-public-testnet.md

```diff
@@ -267,7 +273,16 @@ Then, you will pick a set of video profiles that you would like your input video
 
 ### Broadcasting video
 
-See the [broadcasting guide](https://livepeer.readthedocs.io/en/latest/broadcasting.html) for information on sending video into your broadcaster node and viewing the output transcoded video.
+See the [broadcasting guide](https://livepeer.readthedocs.io/en/latest/broadcasting.html) for information on sending video into your broadcaster node and viewing the output transcoded video. In order to see more detailed logs, run your broadcaster with `-v 99` to enable verbose logging.
+
+After receiving a stream, your broadcaster will try to connect to a set of orchestrators (ether based on the registered orchestrators on-chain or based on the orchestrators specified using the `-orchAddr` flag). Your broadcaster might reject certain orchestrators based on their required price or ticket parameters. 
+
+If you observe the `ticket faceValue higher than max faceValue` error in the broadcaster's logs, you can try:
+
+1. Running the broadcaster with `-depositMultiplier 1`. The default value is 1000 (which is pretty high) meaning that the broadcaster will not willing to use a ticket `faceValue` that exceeds the broadcaster's deposit divided by 1000. So, if the value is 1 then the broadcaster will be willing to use a ticket `faceValue` that equals the broadcaster's deposit. While this might not be desirable in other circumstances, it can be fine for testing purposes
```

Comment:
will not **be** willing to use a ticket...

Reply:
Fixed